app/actions/apps.ts

```diff
@@ -31,7 +31,7 @@ export function doAppCall<Res=unknown>(call: AppCall, intl: any): ActionFunc {
                 }
                 return {data: res};
             case AppCallResponseTypes.ERROR:
-                return {data: res};
+                return {data: res, error: res.error};
```

Comment:
Is this a change we want to follow in all cases both here and in the webapp? There are many other situations where we get an error and we should return error. Not sure what is the best approach.

Reply:
I removed this change. There is a `catch` block below that is returning `error`, but no caller will be looking for that.